ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,125 @@
+                                    sling:orderBefore="placeholder"
+                                    sling:resourceType="fd/af/authoring/components/granite/form/formsmultifield"
+                                    fieldDescription="Enter Display Values (EnumNames) corresponding to their respective Save values"
+                                    fieldLabel="Display Values - EnumNames">
+                                <field
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                        name="./enumNames"
+                                        required="{Boolean}false"/>
+                            </enumNames>
+                            <multiSelect
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                    name="./multiSelect"
+                                    text="Allows multiple selection"
+                                    value="true"/>
+                            <defaultValue
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldDescription="Must be one of the options."
+                                    fieldLabel="Default Value"
+                                    name="./default"/>
+                            <multiValuePath
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:orderBefore="visible"
+                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                    fieldLabel="Items Load Path"
+                                    name="./optionsLoadPath"/>
+                        </items>
+                    </basic>
+                    <help
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Help Content"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured"/>
+                    </help>
+                    <accessibility
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Accessibility"
+                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured"/>
+                    </accessibility>
+                </items>
```

Comment:
also multiselect attribute must be there in the dialog.

Reply:
I'll add this property back